docs/src/App.js

```diff
@@ -0,0 +1,97 @@
+import './App.css';
+import {useState} from "react";
+import DeleteCompletedAlert from "./DeleteCompletedAlert";
+import ListItems from "./ListItems";
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+
+function App(props) {
+    const [items, setItems] = useState(props.initialData);
+    const [completedItems, setCompletedItems] = useState([]);
+    const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+    // const deletedItems = [];
+    const [totalItems, setTotalItems] = useState(props.initialData);
+
+    function handleEditItem(itemId, value, field) {
+        setItems(
+            items.map(
+                (item) => (item.id === itemId ? {...item, [field]: value} : item)
+            )
+        );
+        setTotalItems(items)
+    }
+
+    function handleAddItem(e) {
```

Comment:
Better to have handleAddItem not take an event as a parameter. Instead, only call this function if something should be added (and don't mess with target.value here).

Reply:
Not sure why this method is problematic. Assuming it is, not sure how to add on enter without passing in the event. Are you suggesting the event should only be passed inline?